content/terraform-docs-common/redirects.jsonc

```
@@ -761,5 +735,596 @@
+    "destination": "/terraform/cloud-docs/workspaces/policy-enforcement/define-policies/opa",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cloud-docs/sentinel/import/tfrun",
+    "destination": "/terraform/cloud-docs/workspaces/policy-enforcement/import-reference/tfrun",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cloud-docs/workspaces/policy-enforcement/sentinel/vcs",
+    "destination": "/terraform/cloud-docs/workspaces/policy-enforcement/manage-policy-sets/sentinel-vcs",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cloud-docs/sentinel/mock",
+    "destination": "/terraform/cloud-docs/workspaces/policy-enforcement/test-sentinel",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cloud-docs/workspaces/policy-enforcement/sentinel/mock",
+    "destination": "/terraform/cloud-docs/workspaces/policy-enforcement/test-sentinel",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cloud-docs/workspaces/policy-enforcement/sentinel/json",
+    "destination": "/terraform/cloud-docs/workspaces/policy-enforcement/view-results/json",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cloud-docs/workspaces/run",
+    "destination": "/terraform/cloud-docs/workspaces/run/manage",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cloud/run/manage",
+    "destination": "/terraform/cloud-docs/workspaces/run/manage",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cloud/run",
+    "destination": "/terraform/cloud-docs/workspaces/run/remote-operations",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cloud/workspaces/notifications",
+    "destination": "/terraform/cloud-docs/workspaces/settings/notifications",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cloud/workspaces/run-triggers",
+    "destination": "/terraform/cloud-docs/workspaces/settings/run-triggers",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cloud/workspaces/state",
+    "destination": "/terraform/cloud-docs/workspaces/state",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/intro/v:version(1\\.(?:1|2|3|4|5|6|7|8)\\.x)/phases/:slug*",
+    "destination": "/terraform/intro/phases/:slug*",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/language/index.html",
+    "destination": "/terraform/language",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/language/v:version/index.html",
+    "destination": "/terraform/language",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/configuration/backend",
+    "destination": "/terraform/language/state/backends",
+    "permanent": true
+  },
+  // For 1.1-1.8: backward redirect (new path → old path)
+  {
+    "source": "/terraform/language/v:version(1\\.(?:1|2|3|4|5|6|7|8)\\.x)/backend/:slug*",
+    "destination": "/terraform/language/v:version/settings/backends/:slug*",
+    "permanent": true
+  },
```

```suggestion
  {
    "source": "/terraform/language/v:version(1\\.(?:1|2|3|4|5|6|7|8)\\.x)/backend/:slug*",
    "destination": "/terraform/language/v:version/settings/backends/:slug*",
    "permanent": true
  },
```
I *think* you also need this one without the slug because, for example,  `/terraform/language/v1.8.x/settings/backends`, isn't redirecting

---

Actually, I don't know why this one isn't working. The preview is redirecting some paths on the segment before the slug but not others.